src/util/Constants.js

```diff
@@ -828,9 +828,10 @@ exports.InteractionResponseTypes = createEnum([
  * The type of a message component
  * * ACTION_ROW
  * * BUTTON
+ * * SELECT_MENU
```

Comment:
Is this what Discord calls them?

Reply:
Yeah, "Select Menu" is the term used in the docs provided